pinot-common/src/main/java/org/apache/pinot/common/response/ServerInstance.java

```diff
@@ -140,8 +110,12 @@ public int getPort() {
     return _port;
   }
 
-  public InetAddress getIpAddress() {
-    return _ipAddress;
+  public ServerInstance withSeq(int seq) {
+    return new ServerInstance(_hostname, _port, seq, _shortHostName);
+  }
+
+  public String getShortHostName() {
```

Comment:
It'd be good to put all the getter methods together.

Reply:
Done